### ISSUE TITLE
Team TA -Week 4 : Evaluate your results on validation set and make confusion Matrix.

### ISSUE THREAD

**Issue by uroojbadar** [CubeStatistica/Professional-Training-for-Data-Science-C1#43]
1. Evaluate your model performance on the validation set.
2. Make a confusion matrix of the results.
3. How is the model performing?
4. Is the accuracy more than 80%?
5. Decide whether you need to tune the hyperparameters of the model which you set to default at the start.

**Comment by uroojbadar** [comment]
- [ ] @Zobia12 
- [ ] @SuneetaDevi 
- [ ] @uroojbadar
